auth/server/base/src/main/java/org/wildfly/security/auth/server/SecurityRealm.java

```diff
@@ -64,6 +65,25 @@ default RealmIdentity getRealmIdentity(Evidence evidence) throws RealmUnavailabl
         return principal == null ? RealmIdentity.NON_EXISTENT : getRealmIdentity(principal);
     }
 
+    /**
+     * Get a handle for to the identity for the given evidence in the context of this security realm. Any validation / name
+     * mapping is an implementation detail for the realm. The identity may or may not exist. The principal obtained is
+     * transformed prior to obtaining the identity. The returned handle <em>must</em> be cleaned up by a call to
+     * {@link RealmIdentity#dispose()}.
+     *
+     * Where this method is used to obtain a {@link RealmIdentity} prior to evidence verification the method
+     * {@link RealmIdentity#getEvidenceVerifySupport(Class, String)} will be used to verify if the identity is usable.
+     *
+     * @param evidence an evidence instance which identifies the identity within the realm (must not be {@code null})
+     * @param principalTransformer a function which defines how the identity is transformed before the identity is obtained
```

Comment:
"before the identity" -> "before the authorization identity"

Reply:
"how the identity is transformed" -> "how the principal is transformed"

Reply:
Fixed